src/test/kotlin/org/rust/ide/wordSelection/RsBlocksAndBodiesSelectionHandlerTest.kt

```diff
@@ -0,0 +1,368 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.wordSelection
+
+class RsBlocksAndBodiesSelectionHandlerTest : RsSelectionHandlerTestBase() {
+    fun `test fn body basic`() = doTestWithTrimmedMargins(
+            """
+            |fn foo() {
```

Comment:
Let's use 1 tab instead of 2 here.

(and I agree that it's reasonable to use `|` margin here; it makes selection tests much readable, without "dangling" `<selections>` in air :)